update/update-from-2.21-to-2.22.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/*
+ * Copyright (C) 2022 Igalia, S.L. <info@igalia.com>
+ *
+ * This file is part of PhpReport.
+ *
+ * PhpReport is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PhpReport is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PhpReport.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+require_once('utils.php');
+
+define('PHPREPORT_ROOT', __DIR__ . '/../');
+define('SQLPATH', PHPREPORT_ROOT . 'sql/update/');
+
+/* These are the sql files that must be executed to prepare DB.
+ *
+ * IMPORTANT: they must be ordered for their proper execution.
+ */
+$sqlFiles = array();
+$sqlFiles[] = SQLPATH . "add-updated-at-to-task.sql";
```

Comment:
I missed this before: I think we should also include `bump-db-version-2-22.sql` here. I wasn't planning to do any new "stable" releases so every change in the DB should bump its version so we can easily track where the DB is with regard to the code.

Reply:
Related: #511.

Reply:
I think it's all good, could you do just this small modification?